packages/@react-stately/radio/src/useRadioGroupState.ts

```diff
@@ -74,7 +74,7 @@ export function useRadioGroupState(props: RadioGroupProps): RadioGroupState  {
   });
 
   let setSelectedValue = (value) => {
-    if (!props.isReadOnly && !props.isDisabled) {
+    if (!props.isReadOnly) {
```

Comment:
the reason for this change is because we need to default select the first available item even if the radio group is disabled. without this change, the selected value would never be committed. 

not sure if there are other repercussions though. tests ran fine, behavior seemed the same but i could be missing something.

Reply:
If we're gonna get rid of it for disabled it seems weird to leave it for read only? Did you look at the blame to see when these were added?

Reply:
oh good point. look's like since radio was implemented that it checked for read only. we added the check for disabled later on in this pr https://github.com/adobe/react-spectrum/pull/1424

Reply:
i think useRadio needs updates to handle this, you could move the whole if to surround them instead of handling it here
https://github.com/adobe/react-spectrum/blob/26b9d5db8a0dea35190f032514143e3b18e67439/packages/%40react-aria/radio/src/useRadio.ts#L74
https://github.com/adobe/react-spectrum/blob/26b9d5db8a0dea35190f032514143e3b18e67439/packages/%40react-aria/radio/src/useRadio.ts#L63